test/main.cpp

```diff
@@ -1,42 +1,25 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include <gtest/gtest.h>
 #include <generators.h>
 #include <iostream>
+// #include "tests.cpp"
+// #include "c_api.cpp"
 
 extern std::unique_ptr<OrtEnv> g_ort_env;
 
-void Test_GreedySearch_Gpt_Fp32_C_API();
-void Test_GreedySearch_Gpt_Fp32();
-void Test_BeamSearch_Gpt_Fp32();
-
-#if USE_CUDA
-void Test_Phi2_Cuda();
-void Test_GreedySearch_Gpt_Cuda();
-void Test_BeamSearch_Gpt_Cuda();
-#endif
-
-int main() {
+int main(int argc, char **argv) {
   std::cout << "Generators Utility Library" << std::endl;
-
-  std::cout << "Initializing OnnxRuntime...";
+  std::cout << "Initializing OnnxRuntime... ";
   std::cout.flush();
   Ort::InitApi();
```

Comment:
This comment is not related to your change, but we should think about how to handle the failures that may come from this function. A common failure is the test program failed to load onnxruntime.dll(or loaded the wrong one). We see such issues quite often. So it's better to have a good error message here. If we do nothing, I think by default it will just crash.

Reply:
Suggestion is to move this into the try block

Reply:
Ort::InitApi() will throw an exception with a message, so it should be within a try block.